README.md

```diff
@@ -25,12 +25,14 @@ Assets should be stored in a folder within that category, for example, `talks/sl
 Asset file names should have enough info in the file name to know what references it (such as the associated talk date for slides) or be in a subfolder for things with a lot of assets (such as the TigerOS project).
 
 Current categories:
-* ~Announcements (`announcements/`)~ (deprecated)
+* Announcements (`announcements/`)
 * Events (`events/`)
-* Meetings & Meetups (`meetings-meetups/`) (note that these pages aren't linked from anywhere other than feeds, they are more for the metadata)
+* Meetings & Meetups (`meetings-meetups/`) (Note:  these are not pages; they are placeholder metadata for the calendar feed and should be deleted when creating the announcement post)
```

Comment:
what does this mean? Does this require additional metadata when creating announcement posts?

Reply:
The "pages" in that folder _should_ be entirely metadata and are meant to be used for the Calendar feed generation only. E.g., meetings that haven't had a topic or speaker determined yet and thus have no announcement nor talk page. These use the same metadata that the rest of the site has available, but tends to utilize some functionality more heavily, e.g. `redirect` page type. Technically you _could_ use these as real pages, but I would discourage that in favor of other sections of the site, namely, announcements, talks, and events.